### PR TITLE
Create 2024-05-seara-paradela.md

### PR DESCRIPTION
**Pull request by claudiobonanno**

### CODE REVIEW

File: content/dai-seminars/2024-05-seara-paradela.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
++++
+abstract = "."
+affiliation = "Universitat Politècnica de Catalunya and University of Maryland"
+categories = ["dai-seminar"]
+date = "2024-05-01 12:39:00"
+end = "2024-06-01 12:38:59"
```
Comment (mseri) [inline content/dai-seminars/2024-05-seara-paradela.md:6]: This is the time of the end of the seminar:
```suggestion
end = "2024-05-08 18:00:00"
```

**Comment by mseri** [comment]
This is good, everything would be in one page.
The alternative could be to create two seminar events, one 16.00-17.00 and one 17.00-18.00, then each of them will have their own calendar event instead.

I think I like what you did here more, even if it means that the titles and speaker names in the calendar event look a bit weirder

File: content/dai-seminars/2024-05-seara-paradela.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+answer questions like:
+- Will the orbits be bounded or can the bodies fall apart (and therefore going to "infinity")?
+- Can one find chaotic motion in the three body problem and therefore unpredictability?
+
+These and related questions will be the main goals of this talk, where we will see how the mathematical tools coming from Dynamical Systems
+can be used to deal with this complex problem.
+In particular, we will prove the existence of of oscillatory motions, that is, orbits which leave every bounded region but which return
+infinitely often to some fixed bounded region.
+
+- - -
```
Comment (mseri) [inline content/dai-seminars/2024-05-seara-paradela.md:34]:
```suggestion
- - -

```

**Comment by mseri** [comment]
I am going to merge this so that you can see what it looks like on the website

**Review by mseri — APPROVED** [review]

**Comment by claudiobonanno** [comment]
It’s ok to have them in one single page. The first talk lasts 45 minutes and the second one 30 minutes, and the idea is that they are strongly related.

I’m waiting to see the file online :-)

Claudio

**Pull request merged**